tests/foreman/ui/test_oscap_tailoringfiles.py

```diff
@@ -184,15 +224,39 @@ def test_negative_associate_tailoring_file_with_different_scap(self):
             2. Upload a Mutually exclusive tailoring file
             3. Associate the scap content with tailoring file
 
-        :CaseAutomation: notautomated
-
         :expectedresults: Association should give some warning
 
         :CaseImportance: Critical
         """
+        tailoring_name = gen_string('alpha')
+        content_name = OSCAP_DEFAULT_CONTENT['rhel_firefox']
+        policy_name = gen_string('alpha')
+        with Session(self) as session:
+            make_oscap_tailoringfile(
+                session,
+                name=tailoring_name,
+                tailoring_path=self.tailoring_path,
+                tailoring_loc=self.loc_name,
+                tailoring_org=self.org_name,
+            )
+            self.assertIsNotNone(
+                self.oscaptailoringfile.search(tailoring_name))
+            self.assertIsNotNone(
+                self.oscapcontent.search(content_name))
+            make_oscappolicy(
+                session,
+                content=content_name,
+                name=policy_name,
+                period=OSCAP_PERIOD['weekly'],
+                profile=OSCAP_PROFILE['common'],
+                tailoring=tailoring_name,
+                tailoring_profile=OSCAP_PROFILE['tailoring_rhel7'],
+                period_value=OSCAP_WEEKDAY['friday'],
+            )
```

Comment:
can we catch the warning message?

Reply:
I have no idea where the error would appear, the bug has moved to backlog , as this is a RFE